packages/protocol/contracts/L1/libs/LibProving.sol

```diff
@@ -102,38 +106,34 @@ library LibProving {
                 ),
                 "L1:anchor:calldata"
             );
-        }
-
-        IProofVerifier proofVerifier = IProofVerifier(
```

Comment:
Because of the config change, we use the `proofVerifier` below in `proveBlock`, so it can not be entirely defined here. I mvoed the defintiion outside, but the resolve inside, which should be OK?